### DISCUSSION TITLE
8회차 스터디 진행 현황

### DISCUSSION THREAD

**Discussion by corock** [cowaine/spring-boot-for-msa#97] [0. Announcements]
### 🤔️ Questions & Answers

|                    Username                     | Questions                                                                                                                              | Answers       | 비고    |
|:-----------------------------------------------:|----------------------------------------------------------------------------------------------------------------------------------------|---------------|-------|
|  [ahngilwoong](https://github.com/ahngilwoong)  | - [[Chapter 08] ORM](https://github.com/cowaine/spring-boot-for-msa/discussions/86)                                                    | #93, #95      | 1 / 2 |
|      [coalong](https://github.com/coalong)      | - [[Chapter 08] Spring Data JPA](https://github.com/cowaine/spring-boot-for-msa/discussions/95)                                        | #81, #89      | 1 / 2 |
|       [CoRock](https://github.com/corock)       | - [[Chapter 08] spring-boot-starter-data-jpa 의존성에는 AOP 가 포함되어 있다?](https://github.com/cowaine/spring-boot-for-msa/discussions/89)      | #83, #86, #87 | 1 / 3 |
|      [ParkJiHo](https://github.com/crupy)       | - [[Chapter 08] OLTP](https://github.com/cowaine/spring-boot-for-msa/discussions/83)                                                   | #89           | 1 / 1 |
|   [Klrisation](https://github.com/bunsung92)    | - [[Chapter 08] @GeneratedValue(strategy=GenerationType.AUTO) 에 대한 의구점](https://github.com/cowaine/spring-boot-for-msa/discussions/93) | #83           | 1 / 1 |
|       [JoisFe](https://github.com/joisfe)       | - [[Chapter 08] 도커 애플리케이션에 관련된 질문](https://github.com/cowaine/spring-boot-for-msa/discussions/87)                                      | -             | 1 / 0 |
| [Youngju Jang](https://github.com/youngju-jang) | - [[Chapter 08] GenerationType 종류 필기](https://github.com/cowaine/spring-boot-for-msa/discussions/81)                                   | -             | 1 / 0 |

### Pull Requests

- 전원 PR 완료

**Comment by ahngilwoong** [comment]
수고하셨습니다~
